File: algo_fundamentals_1/01-setting_swapping/setting_swapping.rb
# Set myNumber to 86.  Set myName to your name.
# now swap myNumber into myName & vice versa.

puts myNumber = 86
puts myName = "Lei"

def swap(name, number)
    init_name = name
    init_num = number.to_s

    puts myName = "my name is: " + init_num
    puts myNumber = "my number is:" + init_name
end

swap(myName, myNumber)

File: algo_fundamentals_1/01-setting_swapping/setting_swapping.js
// Set myNumber to 86.  Set myName to your name.
// now swap myNumber into myName & vice versa.

let myNumber = 86
let myName = "Luke"

const swap = (name, number) => {
    let init_name = name
    let init_num = number.toString()

    myName = "my name is: " + init_num
    myNumber = "my number is: " + init_name

    console.log(myName)
    console.log(myNumber)
}

swap(myName, myNumber)